app/models/katello/host/content_facet.rb

```diff
@@ -29,10 +29,26 @@ class ContentFacet < Katello::Model
       validates_with Validators::ContentViewEnvironmentValidator
 
       def update_repositories_by_paths(paths)
```

Comment:
Quite a bit of behavior in this method - please make sure there's test coverage of your changes